docs/www/quick-start-kubernetes.md

```diff
@@ -93,7 +93,7 @@ You can either create a Kubernetes cluster on your local machine or on a cloud p
 2. Install the Redpanda operator CRDs:
 
     ```
-    kubectl apply -k 'https://github.com/vectorizedio/redpanda/src/go/k8s/config/crd?ref=<latest version>'
+    kubectl apply -k https://github.com/vectorizedio/redpanda/src/go/k8s/config/crd?ref=<latest version>
```

Comment:
NIT: Maybe, it will help end user. This works in mac, but requires `jq`.
```suggestion
    VERSION=`curl -s https://api.github.com/repos/vectorizedio/redpanda/releases/latest | jq -r '.["tag_name"]'`
    kubectl apply -k https://github.com/vectorizedio/redpanda/src/go/k8s/config/crd?ref=$VERSION
```

Reply:
Let's skip it for now